src/components/Pricing/data.ts

```diff
@@ -0,0 +1,499 @@
+        description:
+            'Track and visualize trends in your entire codebase with visualizations that are kept automatically up to date.',
+    },
+    notebooks: {
+        label: 'Notebooks',
+        description: 'Create living documentation that interacts directly with your code.',
+    },
+    codeMonitoring: {
+        label: 'Code monitoring',
+        description: 'Get alerts when changes are made to your codebase.',
+    },
+    comprehensiveApi: {
+        label: 'Comprehensive API',
+        description: 'A secure, robust GraphQL API for your repository and code data.',
+    },
+}
+
+const SECURITY_FEATURE_INFO: Record<string, FeatureInfo> = {
+    ssoSaml: {
+        label: 'SSO/SAML',
+        description: 'Single sign-on user authentication with SAML, OAuth, OpenID Connect, and HTTP auth proxy.',
+    },
+    securityRoles: {
+        label: 'User and admin roles',
+        description:
+            'Allow only certain users (site admins) to view and edit site configuration and repository/code host credentials.',
+    },
+    repoPerms: {
+        label: 'Standard repository permissions',
+        description:
+            'Apply the repository permissions from your code host to restrict which repositories a user can search and browse.',
+    },
+    customPerms: {
+        label: 'Custom repository permissions',
+        description:
+            'Apply the repository permissions from your code host or use the explicit permissions API to restrict which repositories a user can search and browse.',
+    },
+    privateInstance: {
+        label: 'Private instance access',
+        description: 'Your instance is only accessible from allowlisted IPs. ',
+    },
+    analytics: {
+        label: 'In-product analytics',
+        description: 'Understand user engagement and calculate the value of using Sourcegraph.',
+    },
+}
+
+const DEPLOYMENT_FEATURE_INFO: Record<string, FeatureInfo> = {
+    cloudDeployment: {
+        label: 'Secure and single-tenant Cloud deployment',
+        description: 'Your own dedicated, single-tenant Sourcegraph instance.',
+    },
+    selfDeployment: {
+        label: 'Self-hosted deployment',
+        description: 'Deploy with Docker, Docker Compose, or Kubernetes on your own infrastructure.',
+    },
+}
+
+export const SPOTLIGHT_FEATURE_INFO: Record<string, FeatureInfo> = {
+    ...CIP_FEATURE_INFO,
+
+    // Code host integrations
+    cloudHosts: {
+        label: 'Unlimited standard Cloud code hosts',
+        description: 'Integrate with GitHub.com, GitLab.com, and Bitbucket Cloud.',
+    },
+    unlimitedCodeHosts: {
+        label: 'Unlimited code hosts',
+        description: 'Search, understand, fix, and automate across multiple code hosts.',
+    },
+    selfHostedCodeHosts: {
+        label: 'Connect to self-hosted code hosts',
+        description:
+            'Integrate with Github Enterprise Self-hosted, GitLab Self-hosted, Bitbucket Server/Data Center, and Perforce.',
```

Comment:
* "GitHub Enterprise Self-hosted" is not the correct name. It's "GitHub Enterprise Server".
* GitLab calls their self-hosted deployments "self-managed". And the "S" should not be capitalized.